test_image_cleanup.py

```diff
@@ -95,3 +99,42 @@ def test_parse_tags():
     assert parse_tags(bad_tags) == False
     assert parse_tags([]) == False
     assert parse_tags({}) == False
+
+
+@patch("builtins.print")
+def test_deregister_loop(mocked_print):
+    test_image = ExampleImage(
+        1, "May 1, 2020 at 10:19:24 AM UTC-4", "atg-test1-123423543"
+    )
+    deregister_loop([test_image], [], True)  # "call" once
+    deregister_loop([test_image], [], False)  # "call" again
+    deregister_loop([test_image], [1], False)  # "call" again, but shouldn't do anything
+    assert mocked_print.mock_calls == [
+        call("1  atg-test1-123423543  May 1, 2020 at 10:19:24 AM UTC-4"),
+        call("This is where I would image.deregister() for 1"),
```

Comment:
Nice use of `unittest.mock.call` helper to simplify the assertions on the call args (I'll have to remember that one).